src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -159,13 +160,63 @@
           />
   </Target>
 
+  <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+    <CommonPriFile Condition="'$(CommonPriFile)' == ''">$(RuntimePath)/resw/resources.pri</CommonPriFile>
+    <TestSpecificPriFile>$(TestPath)\resources.pri</TestSpecificPriFile>
+  </PropertyGroup>
+
+
+
+  <Target Name="CalculateResWFiles" Condition="'$(BuildingUAPVertical)' == 'true'">
```

Comment:
Fair I can see an argument that more resw files will be created during this build invocation.